clientv3/watch.go

```diff
@@ -49,6 +49,11 @@ type Watcher interface {
 	Close() error
 }
 
+type CreatedNotifier interface {
```

Comment:
Oh. I was not aware that the watchrequest is already saved as initReq in the stream struct. If regular watcher wants this, probably it worth the effort to do per watcher one. I will make the change.